### PR TITLE
Getting Started content

### PR DESCRIPTION
**Pull request by andrewazores**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -46,3 +51,95 @@ metadata:
 spec:
   minimal: false
 ```
+
+Or, create the resource graphically in the OperatorHub UI:
```
Comment (ebaron) [inline index.md:55]: Perhaps we should mention that this option is only available if you installed the operator using the bundle image.

Comment (andrewazores) [reply index.md]: Good idea, that could be confusing.

**Review by ebaron — DISMISSED** [review]
Looks good to me!

File: index.md [hunk 2]
```diff
@@ -46,3 +51,96 @@ metadata:
...
+applications, feel free to continue to the next step.
+
+```bash
+$ oc new-app --docker-image=quay.io/andrewazores/quarkus-test:0.0.2
+$ oc patch svc/quarkus-test -p '{"spec":{"$setElementOrder/ports":[{"port":9096},{"port":9999}],"ports":[{"name":"jfr-jmx","port":9096}]}}'
+```
+
+This is a Quarkus container in JVM mode with JMX enabled and pre-configured to
+listen on port 9096.  After deploying the container we patch its service to
+name the 9096 service port `jfr-jmx`, which Cryostat will detect and use to
```
Comment (jiekang) [inline index.md:96]: s/, which Cryostat will/. Cryostat will

File: index.md [hunk 3]
```diff
@@ -46,3 +51,96 @@ metadata:
...
+determine that this is a compatible application that it should pay attention to.
```
Comment (jiekang) [inline index.md:97]: s|compatible application that it should pay attention to.| compatible Java application that it should monitor.

File: index.md (hunk 2 above)
Comment (andrewazores) [reply index.md]: Sounds good.

File: index.md [hunk 4]
```diff
@@ -93,8 +93,8 @@ $ oc patch svc/quarkus-test -p '{"spec":{"$setElementOrder/ports":[{"port":9096}
 
 This is a Quarkus container in JVM mode with JMX enabled and pre-configured to
 listen on port 9096.  After deploying the container we patch its service to
-name the 9096 service port `jfr-jmx`, which Cryostat will detect and use to
-determine that this is a compatible application that it should pay attention to.
+name the 9096 service port `jfr-jmx`. Cryostat will detect and use to
```
Comment (jiekang) [inline index.md:96]: Er, `detect and use this port`.

Comment (jiekang) [reply index.md:96]: or just `use this port` maybe.

**Review by jiekang — APPROVED** [review]

**Pull request merged**